tests/model/test_model.py

```diff
@@ -0,0 +1,68 @@
+# isort: skip_file
+
+import json
+import unittest
+
+import numpy as np
+import pandas as pd
+from sklearn.metrics import mean_absolute_error, mean_squared_error, r2_score  # type: ignore [import-untyped]
+from sklearn.model_selection import train_test_split  # type: ignore [import-untyped]
+
+from src.model.model import Model
+
+
+class TestModel(unittest.TestCase):
+    FEATURE_COLUMNS = [
+        'altitude',
+        'runways',
+        'total_runway_length',
+        'total_arrivals',
+        'total_departures',
+    ]
+    TARGET = ['air_quality']
+
+    def setUp(self) -> None:
+        self.model = Model()
+        self.data_path = 'data/unit_tests/model/mock_airport_dataset.json'
+        self.model_path = 'src/model/model_new.pickle'
```

Comment:
Perhaps we could name this something more sensible to make it clear it is to be used for tests e.g. `random_forest_test_model` or something?